docs/Anwendungskennzeichen.adoc

```diff
@@ -0,0 +1,28 @@
+= Anwendungskennzeichen
```

Comment:
Haben wir bisher docs in .adoc veröffentlicht?
Aktuell sind alle Informationen im Implementation Guide enthalten und das würde ich auch wärmstens empfehlen, dass wir das beibehalten, damit Informationen nicht verteilt sind.
Oder eben die IG Infos ins GitHubRepo ziehen.

Reply:
Lasst uns diese Ausarbeitung als Zuarbeitung des KIMs Teams sehen. Wir sind dafür verantwortlich es an die richtige Stelle im IG zu bringen. Daher Vorschlag, erstmal mergen und nach Übertrag in den IG gerne Löschen um "Single source of truth" zu haben.